plugins/languages/rust/rustaceanvim/settings-options.nix

```diff
@@ -257,8 +257,7 @@ with lib;
     settings =
       helpers.mkNullOrStrLuaFnOr
         (types.submodule {
-          # options = import ../../../lsp/language-servers/rust-analyzer-config.nix lib pkgs;
-          options = { };
+          options = import ../../../lsp/language-servers/rust-analyzer-config.nix lib;
```

Comment:
If `rust-analyzer-config.nix` was a module, then this could be:

```nix
imports = [ ../../../lsp/language-servers/rust-analyzer-config.nix ];
```

And `lib` would be provided by evalModules without needing to import and pass in lib

Reply:
Yeah but this won't be as pretty for the other includes (rust-tools & rustacean.nvim). I'm not sure we would really be gaining anything